app/markets/ActivityTable.tsx

```diff
@@ -96,6 +128,9 @@ export const ActivityTable = ({ id }: { id: string }) => {
                     >
                       {shortenAddress(activity.creator.id)}
                     </a>
+                    {getIsAIAgent(activity.creator.id) && (
+                      <Image src="/ai.svg" alt="ai" width={18} height={18} />
+                    )}
```

Comment:
WDYT about one of these approaches? The idea is to align a bit the elements in the activity table (UI-wise). I forced the AI tag to display conditionally for these examples purposes only.

AI tag first, conditionally display an empty state if no tag (needs an empty div to fill the column):
```tsx
<TableCell className="grid w-1/3 grid-cols-2 items-center gap-2 pl-4 text-text-high-em">
  {getIsAIAgent(activity.creator.id) && idx % 2 === 0 ? (
    <Image src="/ai.svg" alt="ai" width={18} height={18} />
  ) : (
    <div />
  )}
  <a
    href={getGnosisAddressExplorerLink(activity.creator.id)}
    className="hover:underline"
    target="_blank"
    rel="noopener noreferrer"
  >
    {shortenAddress(activity.creator.id)}
  </a>
</TableCell>
```
<img width="540" alt="Screenshot 2024-07-16 at 11 28 46" src="https://github.com/user-attachments/assets/e19645d8-a213-43b4-a551-63fdfcb53131">


Current order, but using columns to display the cell elements (doesn't need an empty div):
```tsx
<TableCell className="grid grid-cols-2 items-center gap-2 pl-4 text-text-high-em">
  <a
    href={getGnosisAddressExplorerLink(activity.creator.id)}
    className="hover:underline"
    target="_blank"
    rel="noopener noreferrer"
  >
    {shortenAddress(activity.creator.id)}
  </a>
  {getIsAIAgent(activity.creator.id) && idx % 2 === 0 && (
    <Image src="/ai.svg" alt="ai" width={18} height={18} />
  )}
</TableCell>
```
<img width="520" alt="Screenshot 2024-07-16 at 11 37 30" src="https://github.com/user-attachments/assets/917f8023-0268-4327-ae58-c08589b62c11">

Reply:
This seems to be tacking a lot of space. Also it breaks a bit on mobile. I think we can take the current solution for now and check if we can improve later.

Reply:
But otherwise it looks good, I like the tabular style.

Reply:
"AI" on the left doesn't work in this case @ElRodrigote cause it's something that's only present in some cases, not always, in that case right is more appropriate otherwise you'll get a bunch of empty space.